src/core/varlink.c

```diff
@@ -424,7 +425,13 @@ static int manager_varlink_init_system(Manager *m) {
                         if (!fresh && varlink_server_contains_socket(m->varlink_server, address))
                                 continue;
 
-                        r = sd_varlink_server_listen_address(m->varlink_server, address, 0666 | SD_VARLINK_SERVER_MODE_MKDIR_0755);
+                        if (path_is_absolute(address)) {
```

Comment:
Could you add a short FIXME comment that SD_VARLINK_SERVER_MODE_MKDIR_0755 flag does not set/fix/update selinux label.

I guess we need to move mkdir-label.[ch] to src/basic/ with using label_ops provided by label.h. But, that's too large and broad change for after RC releases. Hence I think it is better to workaround the issue now in this way, and fix it in more comprehensive way after v258.